base-image/test-files/README.md

```diff
@@ -0,0 +1,26 @@
+# Transformer Unit Tests
+This folder holds the unit tests [TravisCI](https://travis-ci.org/) will run in it's build. These are basic functional unit tests, checks for output formatting and typing. All tests are written in and utilizing [pytest](https://docs.pytest.org/en/latest/). In addition to this, [pylint](https://www.pylint.org/) will also be deployed, look [here](https://github.com/AgPipeline/Organization-info) for our Organization's pylintrc protocols.
```

Comment:
We have a standard for placing a single sentence on its own line. As long as there's not intervening empty lines the parser will join the lines into a paragraph. Please change to one sentence per line